### PR TITLE
add serde_interop quickcheck test

### PR DESCRIPTION
**Pull request by wngr**
.. and handle skipping tags

### CODE REVIEW

File: dag-cbor/tests/serde_interop.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+#[test]
+// TODO: is this fine to fail?
+fn repro() {
+    assert!(raw_serde_roundtrip(ValueArb(Value::Tag(
+        0,
+        Box::new(Value::Bool(false))
+    ))));
+}
```
Comment (wngr) [inline dag-cbor/tests/serde_interop.rs]: Not sure about that.

File: dag-cbor/tests/serde_interop.rs [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+            Value::Integer(if sgn { v } else { -v })
+        }
+        3 => Value::Float(f64::arbitrary(g)),
+        4 => Value::Bytes(Arbitrary::arbitrary(g)),
+        5 => Value::Text(Arbitrary::arbitrary(g)),
+        // recursive variants
+        6 => Value::Array(gen_vec(g, depth - 1)),
+        7 | 8 => Value::Map(gen_map(g, depth - 1)),
+        // TODO: What are the expected semantics for tags?
+        // 8 => Value::Tag(u64::arbitrary(g), Box::new(gen_value(g, depth - 1))),
```
Comment (wngr) [inline dag-cbor/tests/serde_interop.rs]: and here

**Review by wngr — COMMENTED** [review]
Seems to work (but for arbitrary tags)

**Review by dvc94ch — APPROVED** [review]
this only handles it in skip one, I guess that is sufficient for actyx?

**Comment by wngr** [comment]
Yes, that's sufficient.

**Pull request merged**